packages/blockchain-wallet-v4-frontend/src/data/components/buySell/sagas.ts

```diff
@@ -667,7 +667,6 @@ export default ({ api, coreSagas, networks }: { api: APIType; coreSagas: any; ne
   const fetchSDDEligible = function* () {
     try {
       yield put(A.fetchSDDEligibleLoading())
-      yield call(waitForUserData)
```

Comment:
why is this removed?

Reply:
I think the `isTier2` call invoke the `waitForUserData`